<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="warn_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="avatar_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ff781f;&quot;&gt;头像&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 120, 31);
border-image: url(:/images/images/avatar-border.svg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/avatar.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="login_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ff781f;&quot;&gt;登录&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    color: white;
    font: bold 13pt;
    border:none;
    min-height: 24px;
    min-width: 60px;
	border-image: url(:/images/images/button-login.svg);
	
}
QPushButton:hover{
    color: lightgray;
    border:none;
	border-image: url(:/images/images/button-login-hover.svg);
}
QPushButton:pressed {
    color: lightgray;
    border:none;
	border-image: url(:/images/images/button-login-press.svg);
    padding-top: -15px;
    padding-bottom: -17px;
} </string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="register_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ff781f;&quot;&gt;点击注册&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#FF781F;&quot;&gt;注册帐号&amp;gt;&amp;gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
			border-image: url(./images/button-quit.svg);
   			border:none;
}
QPushButton:hover{border:none;border-image: url(./images/button-quit.svg);}
QPushButton:pressed{border:none;border-image: url(./images/button-quit.svg);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/button-quit.svg</normaloff>:/images/images/button-quit.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Min_pushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-image: url(./images/button-quit.svg);
   border:none;
}
QPushButton:hover{border:none;border-image: url(./images/button-quit.svg);}
QPushButton:pressed{border:none;border-image: url(./images/button-quit.svg);}
   </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/button-minimize.svg</normaloff>:/images/images/button-minimize.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QMenuButton" name="Status_pushButton">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>165</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    border:none;
}
QPushButton:hover{
    border:none;
}
QPushButton:pressed {

    border:none;
} </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/status-online-1.svg</normaloff>:/images/images/status-online-1.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#ff781f;&quot;&gt;帐号：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="buddy">
       <cstring>userid_comboBox</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="userid_comboBox">
      <property name="styleSheet">
       <string notr="true">QComboBox {
    
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}

QComboBox:editable {
    background: white;
}

QComboBox:!editable, QComboBox::drop-down:editable {
		border:none;
     background: white;
}

/* QComboBox gets the &quot;on&quot; state when the popup is open */
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: white;
	border:none;
}

QComboBox:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {

	image: url(:/images/images/unfold_icon.png);
}

QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 10px;
}</string>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#FF781F;&quot;&gt;密码：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="buddy">
       <cstring>password_text</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="password_text">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ff781f;&quot;&gt;密码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>188</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="RemeberPwd_checkBox">
      <property name="styleSheet">
       <string notr="true">QCheckBox {
    spacing: 10px;
	color: rgb(255, 120, 31);
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
}

QCheckBox::indicator:unchecked {
   
	image: url(:/images/images/checkBox-unselect.svg);
}

QCheckBox::indicator:unchecked:hover {
 
	image: url(:/images/images/checkBox-unselect.svg);
}

QCheckBox::indicator:unchecked:pressed {
    
	image: url(:/images/images/checkBox-unselect.svg);
}

QCheckBox::indicator:checked {
   
	image: url(:/images/images/checkBox-select.svg);
}

QCheckBox::indicator:checked:hover {
    
	image: url(:/images/images/checkBox-hover.svg);
}

QCheckBox::indicator:checked:pressed {
    
	image: url(:/images/images/checkBox-hover.svg);
}

QCheckBox::indicator:indeterminate:hover {
    
	image: url(:/images/images/checkBox-hover.svg);
}

QCheckBox::indicator:indeterminate:pressed {
  image: url(:/images/images/checkBox-hover.svg);
}</string>
      </property>
      <property name="text">
       <string>记住密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="AutoLogin_checkBox">
      <property name="styleSheet">
       <string notr="true">QCheckBox {
    spacing: 10px;
	color: rgb(255, 120, 31);
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
}

QCheckBox::indicator:unchecked {
   
	image: url(:/images/images/checkBox-unselect.svg);
}

QCheckBox::indicator:unchecked:hover {
 
	image: url(:/images/images/checkBox-unselect.svg);
}

QCheckBox::indicator:unchecked:pressed {
    
	image: url(:/images/images/checkBox-unselect.svg);
}

QCheckBox::indicator:checked {
   
	image: url(:/images/images/checkBox-select.svg);
}

QCheckBox::indicator:checked:hover {
    
	image: url(:/images/images/checkBox-hover.svg);
}

QCheckBox::indicator:checked:pressed {
    
	image: url(:/images/images/checkBox-hover.svg);
}

QCheckBox::indicator:indeterminate:hover {
    
	image: url(:/images/images/checkBox-hover.svg);
}

QCheckBox::indicator:indeterminate:pressed {
  image: url(:/images/images/checkBox-hover.svg);
}</string>
      </property>
      <property name="text">
       <string>自动登录</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>297</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="title_label">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>images/title.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="logo_label">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>images/logo.png</pixmap>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>warn_label</zorder>
  <zorder>avatar_label</zorder>
  <zorder>login_button</zorder>
  <zorder>register_label</zorder>
  <zorder>pushButton</zorder>
  <zorder>Min_pushButton</zorder>
  <zorder>Status_pushButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QMenuButton</class>
   <extends>QPushButton</extends>
   <header location="global">qmenubutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
